Project.toml

```diff
@@ -19,6 +20,7 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 RandomExtensions = "fb686558-2515-59ef-acaa-46db3789a887"
 RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
 Singular = "bcd08a7b-43d2-5ff7-b6d4-c458787f915c"
+SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
```

Comment:
I am somewhat reluctant to add a depdency on `SparseArrays` as we have our own sparse matrix type. But I guess it depends on what you use it for, which is not clear to me.

Reply:
This changed now and the code uses only the Oscar-Type now. But there are operations that are not supported in Oscar, such as SparseMatrix*SparseMatrix and the Kronecker-Product.

Reply:
I am confused; what does "This changed now" mean? `SparseArrays` still is listed here (as is `Distributed`)